parser/SqlParser.ypp

```diff
@@ -440,6 +446,9 @@ void NotSupported(const YYLTYPE *location, yyscan_t yyscanner, const std::string
   table_constraint_def  // unimplemented
   table_constraint_def_commalist        // unimplemented
   opt_table_constraint_def_commalist    // unimplemented
+  opt_column_list       // unimplemented
+  index_type            // unimplemented
```

Comment:
I think it makes sense for the type of this to be the enum `IndexSubBlockType` from `storage/StorageBlockInfo.hpp`